test.md

```diff
@@ -0,0 +1 @@
+"This file should be ignored by git".
```

Comment:
- [ ] Let's ensure this file dissapears from our repository by adding it to `.gitignore`